Cargo.toml

```
@@ -32,7 +32,7 @@ assert_cmd = "2.0.4"
 nix = { version = "0.26.1", features = ["signal"] }
 tempfile = "3.2.0"
 tokio = "1.24.2"
-substrate-rpc-client = { git = "https://github.com/paritytech/substrate", branch = "master" }
+substrate-rpc-client = { git = "https://github.com/davxy/substrate", branch = "move-beefy-to-consensus" }
```

this should definitely not change

```suggestion
substrate-rpc-client = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Same for all `Cargo.toml` files